src/test/java/TodoTest.java

```diff
@@ -0,0 +1,18 @@
+//package duke;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+    @Test
+
+    public void getTypeTest() {
+        Todo todo = new Todo("test");
+        assert(todo.getType() == "T");
```

Comment:
I really like how you did testing of the lowest complexities as it shows that you place focus on the minute details of your code!